### PR TITLE
Renamings to do with the stopgap interpreter.

### LINKED ISSUE #220: ykbh
We have an internal crate `ykbh` and a test `ykbh.rs` -- but I'm not sure what `bh` stands for, and it's not documented. Can we clarify this somewhere? [Or, if I missed it, point me in the right direction ;)]

### PR DESCRIPTION
**Pull request by vext01**
This commit does several (mostly mechanical) changes:

 - ykbh -> yksg.
 - SIRInterpreter -> StopgapInterpreter.
 - Updated mod-level docstring in yksg.
 - blackhole -> stopgap.

Fixes #220

### CODE REVIEW

File: internal_ws/yksg/src/lib.rs [hunk 1]
```diff
@@ -1,6 +1,9 @@
-//! The stopgap interpreter which is invoked during a guard failure. It picks up right after the
-//! guard and interprets SIR getting us safely back to a control point in the the meta-tracer from
-//! which point normal interpretation can continue.
+//! The stopgap interpreter.
```
Comment (vext01) [inline internal_ws/yksg/src/lib.rs:1]: This is the only non-mechanical change here.

File: internal_ws/yksg/src/lib.rs [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
+//!
+//! This is a SIR interpreter which is invoked after a guard failure. It interprets from the point
+//! of divergence in the compiled trace up until the next control point. Once arrived at the
```
Comment (vext01) [inline internal_ws/yksg/src/lib.rs:4]: Oh, it might help to define "compiled" here...

Comment (ltratt) [reply internal_ws/yksg/src/lib.rs]: The current comment is a bit hard to parse. I think you can simplify it to something like "After a guard failure, the StopgapInterpreter takes over execution. It interprets SIR to execute the program from the guard failure until execution arrives back to the control point, at which point the normal interpreter can continue."

Comment (vext01) [reply internal_ws/yksg/src/lib.rs]: Fixed in 55137d0cb99e3c6dabb17720a353398f600e196e.

**Comment by ltratt** [comment]
Please squash.

**Comment by vext01** [comment]
splat.

**Comment by vext01** [comment]
formatted.

**Comment by ltratt** [comment]
Please squash.

**Comment by vext01** [comment]
splat.
